persistence/pre_persistence/service_node_test.go

```diff
@@ -136,7 +137,7 @@ func TestDeleteServiceNode(t *testing.T) {
 	ctx := NewTestingPrePersistenceContext(t)
 	actor := NewTestServiceNode()
 	if err := ctx.InsertServiceNode(actor.Address, actor.PublicKey, actor.Output, actor.Paused, int(actor.Status),
-		actor.ServiceURL, actor.StakedTokens, actor.Chains, int64(actor.PausedHeight), actor.UnstakingHeight); err != nil {
+		actor.ServiceUrl, actor.StakedTokens, actor.Chains, int64(actor.PausedHeight), actor.UnstakingHeight); err != nil {
```

Comment:
This function's argument list is quite long imo. Could benefit from a struct as an argument in my opinion, wdyt?

Reply:
One case where a long argument list would be cumbersone is when changing the signature of the function either by adding or removing one argument. The situation would be far to easy when using a struct.

Candidates: `ServiceNodeProps{}`